docs/configuration/identity-providers/oidc.md

```diff
@@ -136,6 +136,26 @@ The private key in DER base64 encoded PEM format used to encrypt the [OpenID Con
 
 Can also be defined using a [secret](../secrets.md) which is the recommended for containerized deployments.
 
+### access_token_lifespan
+
+The maximum lifetime of an access token.
+
+### authorize_code_lifespan
+
+The maximum lifetime of an authorize code.
```

Comment:
We should probably add some recommendations for each of them to help people.

Reply:
Done.